qutebrowser/misc/lineparser.py

```diff
@@ -289,3 +305,7 @@ def save(self):
         with qtutils.savefile_open(self._configfile, self._binary) as f:
             self._write(f, self.data[-limit:])
         self._after_save()
+
+    def clear(self):
+        self.data = []
+        self.save()
```

Comment:
This isn't needed as it's inherited from `LineParser`, but I'll take care of it.